src/mono/wasm/Wasm.Build.Tests/Blazor/IcuTests.cs

```diff
@@ -0,0 +1,132 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+#nullable enable
+
+namespace Wasm.Build.Tests.Blazor;
+
+// these tests only check if correct ICU files got copied
+public class IcuTests : BlazorWasmTestBase
```

Comment:
What are the tests here testing for? Only the warning? Should they instead try to use the icu data?

Reply:
They test for 1) ICU files copied to AppBundle 2) correct hierarchy of MsBuild flags 3) yes, warnings